server/accounts_test.go

```diff
@@ -1305,12 +1305,6 @@ func TestAccountRequestReplyTrackLatency(t *testing.T) {
 	}
 
 	// Now let's add in tracking
-
```

Comment:
Why did you remove this test?

Reply:
@wallyqs

Reply:
moved the check on whether system account is defined to be done earlier when processing the config at: https://github.com/nats-io/nats-server/pull/1130/files#diff-7b256a4f472c7d4b9e1931f2f1073b66R1462 but will try to add it back

Reply:
I fixed this in JWT branch. Inside of `TrackServiceExportWithSampling` do this..

```go
	if a.srv != nil && !a.srv.eventsEnabled() {
		return ErrNoSysAccount
	}
```

Instead of what is there. On init the a.srv will be nil so skips the test.

Reply:
Yeah, this seems to work!

Reply:
Perfect.